### PR TITLE
call integration test from azure pipeline

### LINKED ISSUE #842: Add integration tests into the Azure Pipeline
#810 needs to be done first.

The work is modify [this line](https://github.com/webhintio/webhint.io/blob/214bffeadca179967e4093bf95b6b42570bcf03b/azure-pipelines.yml#L28) to call also the new script.
Something like `npm run test-staging && node helpers/integration-tests.js`

### PR DESCRIPTION
**Pull request by uchemukolo**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)
Fix #842 

- Calls the integrated tests from azure pipeline

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
       GIT_USER_EMAIL: $(GIT_USER_EMAIL)
       GIT_USER_NAME: $(GIT_USER_NAME)
       GIT_PASSWORD: $(GIT_PASSWORD)
-  - script: npm run test-staging
+  - script: npm run test-staging && node helpers/integration-tests/index.js
```
Comment (molant) [inline azure-pipelines.yml]: @uchemukolo can you add this also to line 19 in a separate commit? That way we can test it on Azure:

`  - script: npm test && node helpers/integration-tests/index.js`

We will have to remove it later but that will help us identify if there are any issues with puppeteer on Azure.

Thank you 🙏

Comment (uchemukolo) [reply azure-pipelines.yml]: This has been don and pushed

**Comment by molant** [comment]
And this is the status of CI 

image

Looks like everything is working  as it should 🎉🎉🎉

Can you remove the part after `npm test` so we can merge it?

Thank you so much!

**Comment by molant** [comment]
Thanks a lot @uchemukolo and congratulations in your first contribution to the webhint project 🎉

**Comment by antross** [comment]
Thanks @uchemukolo, this will be great to help us catch regressions in the site before going to production! We've been burned by this a few times in the past.

**Comment by uchemukolo** [comment]
Glad I could help @molant @antross 
This is my first contribution to open source

**Comment by molant** [comment]
> This is my first contribution to open source

Double congratulations then! Hope it is the first of many. Let us know if there's anything we can do to help!

**Pull request merged**